packages/calcite-design-tokens/.eslintrc.js

```diff
@@ -0,0 +1,156 @@
+module.exports = {
+  root: true,
+  env: {
+    "jest/globals": true
+  },
+  extends: [
+    "plugin:@cspell/recommended",
+    "plugin:@esri/calcite-components/recommended",
+    "plugin:@stencil-community/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "plugin:jsdoc/recommended",
+    "prettier"
+  ],
+  ignorePatterns: ["dist", "docs", "hydrate", "www"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: ["tsconfig-eslint.json"],
+    ecmaVersion: 2021,
+    sourceType: "module"
+  },
+  plugins: [
+    "@esri/calcite-components",
+    "@typescript-eslint",
+    "eslint-plugin-react",
+    "import",
+    "jest",
+    "jsdoc",
+    "prettier",
+    "unicorn"
+  ],
+  rules: {
+    "@esri/calcite-components/ban-events": [
+      "warn",
+      {
+        event: "keyup",
+        message: "Use keydown instead for consistent interaction behavior (e.g., closing, moving focus, etc.)."
+      },
+      {
+        event: "keypress",
+        message: "Use keydown instead for consistent interaction behavior (e.g., closing, moving focus, etc.)."
+      }
+    ],
+    "@stencil-community/decorators-context": "off",
+    "@stencil-community/decorators-style": "warn",
+    "@stencil-community/no-unused-watch": "off",
+    "@stencil-community/own-methods-must-be-private": "off",
+    "@stencil-community/own-props-must-be-private": "off",
+    "@stencil-community/prefer-vdom-listener": "warn",
+    "@stencil-community/required-jsdoc": "off",
+    "@stencil-community/strict-boolean-conditions": "off",
+    "@typescript-eslint/ban-types": "warn",
+    "@typescript-eslint/explicit-module-boundary-types": [
+      "error",
+      {
+        allowArgumentsExplicitlyTypedAsAny: true,
+        allowedNames: [
+          "connectedCallback",
+          "disconnectedCallback",
+          "componentWillRender",
+          "componentDidRender",
+          "componentWillLoad",
+          "componentDidLoad",
+          "componentWillUpdate",
+          "componentDidUpdate",
+          "render"
+        ]
+      }
+    ],
```

Comment:
Can you make sure the design token's eslintrc doesn't contain unnecessary stuff? A lot of this is specific to CC